src/main/java/seedu/address/ui/DetailsPanel.java

```diff
@@ -7,38 +7,49 @@
 import seedu.address.commons.core.DetailsPanelTab;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.PersonEvent;
+import seedu.address.model.person.PersonStreak;
 
 public class DetailsPanel extends UiPart<Region> {
 
     private static final String FXML = "DetailsPanel.fxml";
 
     private final PersonDetailsTab personDetailsTab;
     private final UpcomingEventsTab upcomingEventsTab;
+    private final StreaksTab streaksTab;
 
     @FXML
     private VBox tabPlaceholder;
 
     /**
      * Creates a {@code DetailsPanel} with the given {@code ObservableList}s.
+     *
      * @param upcomingEventsList A list of upcoming events.
-     * @param detailedPerson A list containing a single {@code Person}.
+     * @param detailedPerson     A list containing a single {@code Person}.
      */
-    public DetailsPanel(ObservableList<PersonEvent> upcomingEventsList, ObservableList<Person> detailedPerson) {
+    public DetailsPanel(ObservableList<PersonEvent> upcomingEventsList,
```

Comment:
Remember to update the documentation to include the new `personStreaks` param

Reply:
Will add as an issue first!